themes/stanford_basic/algolia-search/src/styled-components.tsx

```diff
@@ -78,30 +78,32 @@ export const SearchForm = styled.form`
       }
     }
 
-    .open-modal {
-      color: #2E2D29;
-      font-size: 21px;
-      font-weight: 400;
-      text-decoration: none;
-      border-radius: 40px;
-      border: 1px solid #C0C0BF;
-      background: #fff;
-      display: flex;
-      flex-direction: row;
-      padding: .8rem 2rem 1rem;
-
-      &:hover, &:focus {
-        background: transparent;
+    @media ( max-width: 768px) {
+      .open-modal {
```

Comment:
@jenbreese why do you need the media query. The element isn't even visible over 768px, so any styles are only applicable to the smaller sizes anyways. CSS is about keeping things clean.

Reply:
Am I thinking the wrong way?

Reply:
I fixed it for you. see my change: https://github.com/SU-SWS/stanford_profile/pull/970/commits/8ee1d2096e71c77df8972697130a65befb6d6163